xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4471,10 +4471,10 @@ bool CVideoPlayer::IsInMenu() const
   return m_State.isInMenu;
 }
 
-bool CVideoPlayer::HasMenu() const
+MenuType CVideoPlayer::GetSupportedMenuType() const
```

Comment:
generally all implementations of `GetSupportedMenuType()` can be `const` like it is done here.
i'm just not sure how this works with the pure virtual function of the interface:
https://github.com/xbmc/xbmc/blob/74e0e2f390c7e6a4d59a4eea3f7dd00a8649f383/xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStream.h#L113-L117

Reply:
sorry, don't do this ;) it doesnt work.
the sharedptr is enough